src/main/groovy/au/org/ala/ecodata/reporting/ProjectXlsExporter.groovy

```diff
@@ -187,7 +187,7 @@ class ProjectXlsExporter extends ProjectExporter {
         this.projectService = projectService
         this.organisationService = organisationService
         distinctElectorates = new ArrayList()
```

Comment:
This is a MERIT class, once you've deployed this built to staging I'll need to test the download with it.